client/service-container-chain/src/service.rs

```diff
@@ -481,7 +489,7 @@ fn start_consensus_container(
                     })?;
 
                     let authorities = tc_consensus::authorities::<Block, ParachainClient, NimbusPair>(
-                        orchestrator_client_for_cidp.as_ref(),
+                        orchestrator_client_for_cidp.as_ref().unwrap(),
```

Comment:
Is it possible to remove the `unwrap` here?

Reply:
I tried but it was too many changes. This is related to the comment about `BuyCoreParams`, will be fixed in the future.

Reply:
What about `expect` message?

Reply:
*fixed*